common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4421,6 +4421,12 @@
     ],
     "sqlState" : "38000"
   },
+  "RECURSION_LEVEL_LIMIT_EXCEEDED" : {
+    "message" : [
+      "Recursion level limit <levelLimit> reached but query has not exhausted, try increasing CTE_RECURSION_LEVEL_LIMIT"
```

Comment:
shall we mention the config name here? Otherwise this error message is not actionable.